packages/react/src/components/accordion/accordion-styles.ts

```diff
@@ -0,0 +1,51 @@
+import styled, { css } from 'styled-components';
+import { Theme } from '../../themes';
+
+export const StyledAccordionGroup = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+export const AccordionSectionStyled = (isExpanded: boolean, theme: Theme) => css`
+    background: ${theme.greys['colored-white']};
+    border: ${isExpanded ? `1px solid ${theme.greys['dark-grey']}` : '0'};
+    border-radius: 0 0 4px 4px;
+    border-top-width: 0;
+    color: #1b1c1e;
+    font-size: 0.75rem;
+    font-weight: 400;
+    height: ${isExpanded ? 'auto' : '0'};
+    letter-spacing: 0.24px;
+    line-height: 1.7;
+    margin-bottom: 8px;
+    min-height: 0;
+    overflow: hidden;
+    padding: ${isExpanded ? '16px 24px 24px 40px' : '0'};
+    transition: height 300ms cubic-bezier(0.25, 0.46, 0.45, 0.94) 0ms;
+` as any;
+
+export const HeadingStyled = (isExpanded: boolean, theme: Theme) => css`
```

Comment:
Some issues with the margins.
<img width="191" alt="Screenshot 2023-08-21 at 1 27 30 PM" src="https://github.com/kronostechnologies/design-elements/assets/3722754/ad2d5f86-af65-425a-a5c6-180cbc581313">

Reply:
I can't replicate this issue.
I tried on 
chrome: Version 115.0.5790.102 (Official Build) (arm64)
FF: 116.0.3 (64-bit)
safari: Version 16.6 (18615.3.12.11.2)